lib/services/ios-project-service.ts

```diff
@@ -357,6 +357,10 @@ export class IOSProjectService extends projectServiceBaseLib.PlatformProjectServ
 		return this.$fs.exists(path.join(projectRoot, this.$projectData.projectName, constants.APP_FOLDER_NAME));
 	}
 
+	public isPlatformBuilt(projectRoot: string): IFuture<boolean> {
+		return this.$fs.exists(path.join(projectRoot, "build"));
```

Comment:
Check my comment about android-project-service. I can't imagine a situation caused by the CLI in which this folder would be empty. However, if you think that such situation may occur, I could extend the check.